README.md

```diff
@@ -36,6 +36,70 @@ over 100,000 metric values per second with less than 0.2% overhead.
 
 # Building the OVIS / LDMS source code
 
+## Pre-built containers
+
+You may skip the building process and use the pre-built containers from the
+following list. Please follow the corresponding links for more information on
+the images.
+- [ovishpc/ldms-samp](https://hub.docker.com/r/ovishpc/ldms-samp):
+  a small image for 'sampler' daemons meant to be deployed on compute nodes.
+- [ovishpc/ldms-agg](https://hub.docker.com/r/ovishpc/ldms-agg):
+  an image for 'aggregator' daemons which also include various storage plugins.
+- [ovishpc/ldms-storage](https://hub.docker.com/r/ovishpc/ldms-storage):
+  an image that contains storage technologies (e.g. SOS, Kafa).
+- [ovishpc/ldms-web-svc](https://hub.docker.com/r/ovishpc/ldms-web-svc):
+  an image for the back-end (django) that query SOS data for Grafana server.
+- [ovishpc/ldms-grafana](https://hub.docker.com/r/ovishpc/ldms-grafana):
+  a Grafana image with 'DSOS' Grafana plugin that allows Grafana to get data
```

Comment:
with 'DSOS' --> with the 'DSOS'

Reply:
specific name .. do we need an article there? like 'a server with MySQL' we don't say 'a server with the MySQL', right?